src/foundry/diceTerms/rollTerms/poolTerm.d.ts

```diff
@@ -19,11 +19,15 @@ declare class PoolTerm extends RollTerm {
    * @param termData -  Data used to create the Dice Term, including the following:
    *                    (default: `{}`)
    * @param terms - The original provided terms to the Dice Pool
+   *                (default: `[]`)
    * @param modifiers - The string modifiers applied to resolve the pool
+   *                    (default: `[]`)
    * @param rolls - The elements of a Dice Pool must be Roll objects or numbers
+   *                (default: `[]`)
    * @param results - The array of dice pool results which have been rolled
+   *                  (default: `[]`)
```

Comment:
I am realizing only now that the original does not have documented parameters. So the doc strings for them should be removed and only the default remarks kept.